### DISCUSSION TITLE
(*Completely* optional) When time and availability arrive, please rebase FGKASLR to upstream

### DISCUSSION THREAD

**Discussion by KyunLFA** [alobakin/linux#20] [General]
Please don't get me wrong, I'm not demanding or meaning to impose, just expressing my interest in seeing it updated for my own personal use (I don't know enough to do it myself, though I've tried). Even just a status update on how the development is headed would be awesome.

[This commit should help in that](https://github.com/torvalds/linux/commit/ac3b43283923440900b4f36ca5f9f0b1ca43b70e) from my testing of my own hacking on your code to work with upstream.

I'll keep trying to make it work in my own pet project, but I'd be tremendous if you had the opportunity whenever it comes to update it officially and I'd be thrilled to report back anything I find as I'd use that in my own kernel pet project.

Thank you for your amazing contributions!

**Comment by alobakin** [comment]
Hi. Sure. It's just me having too many projects at once :D
Let's see when I could get back to this. It's not currently requested by anyone except my curiosity, so it's not in the first place of the priorities list...
